src/components/filmDetailsPage/MiddleBlock.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="middle-block">
+    <div class="info-block">
+      <Info message="Films by drama genre" />
```

Comment:
Consider creating a "core" folder under "src" where all the main reusable non-react things will be stored.

Under "src/core" folder that you've created consider creating a folder "i18y" which stands for "internationalisation". This folder in real project usually holds translation object in cases when they're stored in the source code. The folder structure consists of "index.js" file and a number of "*.js" files for different languages (e.g. "en.js" for English or "ge.js" for German). Each of these language files contains an object with the same keys but different values for the target language. The idea behind it is to decouple the selected language for a website and the actual text used in all the components.
E.g. your data object will have an text as a field as follows:
`infoMessage: I18Y['EN'].DRAMA_FILMS,` for English and `infoMessage: I18Y['GE'].DRAMA_FILMS,` for German. As you might guess, this would allow to hook up language selection logic from your app that will pass either "EN" or "GE" to the I18Y object, but you the same "DRAMA_FILMS" key - and end up with different language texts displayed on the live website.
For guidance and examples, take a look at the variation of this technique by one of my mentees https://github.com/cqxg/Way-of-the-Samurai/blob/2d715673f9f2ccc2509ba702c869fc8a52f565a5/src/utils/constants.js#L10
In the end, you task is to create such object (could be only English for now) and go though the whole app and change the hard-coded text to the usage of this I18Y object.